Game/Scripts/UITextTrigger.cpp

```diff
@@ -20,41 +30,79 @@ UITextTrigger::~UITextTrigger()
 
 void UITextTrigger::Start()
 {
-	player = App->GetModule<ModulePlayer>()->GetPlayer()->GetComponent<ComponentPlayer>();
 	componentRigidBody = owner->GetComponent<ComponentRigidBody>();
+	input = App->GetModule<ModuleInput>();
+
+	textBoxSize = textBox.size() - 1;
+	displacementControl = textBox[textBoxCurrent]->GetComponent<UIImageDisplacementControl>();
```

Comment:
```suggestion
	displacementControl = textBox[static_cast<size_t>(textBoxCurrent)]->GetComponent<UIImageDisplacementControl>();
```

Avoids raising a warning.